test_api_data.py

```diff
@@ -0,0 +1,50 @@
+import api_data as api
+
+from mock import Mock
```

Comment:
If you use packages for tests, then they should be at least part of some tests requirements file.